daemon/cmd/agenthealth.go

```diff
@@ -33,6 +34,14 @@ func (d *Daemon) startAgentHealthHTTPService() {
 
 	mux := http.NewServeMux()
 	mux.Handle("/healthz", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		includeK8sCheck := false
+		if v := r.Header.Get("include-k8s-check"); v != "" {
+			res, err := strconv.ParseBool(v)
+			if err != nil {
+				log.WithError(err).WithFields(logrus.Fields{"value": v}).Warn("include-k8s-check should be bool")
+			}
+			includeK8sCheck = res
```

Comment:
Nit: I think the return value of the `ParseBool` on error is undefined. I'd set false (default) explicitly on failure or return 400 (Bad Request).